src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java

```diff
@@ -1,16 +1,87 @@
 package com.customer.customerService;
 
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import com.customer.CustomerResponse.CustomerResponse;
+import com.customer.address.Address;
+import com.customer.model.Customer;
+import com.customer.repository.CustomerRepository;
+import com.customer.service.CustomerServiceImpl;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class CustomerServiceApplicationTests {
 
+	@Mock
+	CustomerRepository customerRepository;
+
+	@Mock
+	CustomerResponse customerResponse;
+
+	@InjectMocks
+	CustomerServiceImpl customerServiceImpl;
+
 	@Test
-	public void contextLoads() {
+	public void postRequestTest() {
+		// Given
+		Address address = Address.builder().streetName("Abc").state("Haryana").country("France").build();
+		Customer customerRequest = Customer.builder().firstName("Sita").lastName("Ji").address(address).build();
+		when(customerRepository.save(customerRequest)).thenReturn(customerRequest);
+
+		// When
+		Customer custResponse = customerServiceImpl.createCustomer(customerRequest);
+
+		// Then
+		Assert.assertEquals(customerRequest.getFirstName(), custResponse.getFirstName());
+	}
+
+	@Test
+	public void getRequestTest() {
+		// Given
+		Date date = java.util.Calendar.getInstance().getTime();
+		Calendar c = Calendar.getInstance();
+		Date currentDate = c.getTime();
+		c.setTime(date);
+		c.add(Calendar.YEAR, 2);
+		Date experienceDate = c.getTime();
```

Comment:
For next time, read more into : 
LocalDate
LocalTime
LocalDateTime

Use them instead of Date, as they are the latest Date APIs